### DISCUSSION TITLE
How to programatically disable checkbox options.

### DISCUSSION THREAD

**Discussion by Sbabureddy** [react-hook-form/react-hook-form#11300] [General] [answered]
Hello everyone, I am working on checkboxes, where there was a requirement to disable certain options.

example: if I select A , then B and C should be disabled. can anyone help me to achieve this using `react-hook-form`.

@bluebill1049 

https://codesandbox.io/p/sandbox/register-forked-6x83rs?file=%2Fsrc%2FApp.js%3A49%2C63

**Comment by chanthinh** [comment, marked as answer]
Hi @Sbabureddy,
This is my codesandbox, hope this helps you
https://codesandbox.io/p/sandbox/register-forked-yzrcr7?file=%2Fsrc%2FApp.js%3A31%2C69

**Reply by Sbabureddy** [reply]
Thanks @chanthinh
